spec/requests/submissions_search_spec.rb

```diff
@@ -640,5 +640,28 @@
         expect(laa_references).to match(%w[LAA-CCCCCC LAA-BBBBBB LAA-AAAAAA])
       end
     end
+
+    context "when searching for unsupported queries" do
+      before do
+        create_list(:submission, 2, :with_pa_version, defendant_name: "Joe Bloggs")
+
+        # This travel_to is used to test that "raw data" and [view] data are sync'd by page and order.
```

Comment:
Nit: this comment, and the accompanying block, has been copy-pasted from a test that actually used it, but it's nothing to do with what's being tested here. Can we get rid of everything in this `before` block we don't actually need?